test-workspace/test-crate-custom-lib-name/src/lib.rs

```diff
@@ -0,0 +1,159 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
```

Comment:
Rather than copy the entire test-crate, maybe just pare this crate down to 1 or 2 violations. The test-crate is intended to be the primary test that exercises all the validations, and this one should just test the modified crate name works throughout.

Otherwise, looks good!

Reply:
>  The test-crate is intended to be the primary test that exercises all the validations, and this one should just test the modified crate name works throughout.

Makes sense! Reworked in 7b3f632.